skt/executable.py

```diff
@@ -27,6 +27,8 @@
 import tempfile
 import time
 import traceback
+import atexit
```

Comment:
Can you please keep the import order of standard packages - third party packages - local packages and the alphabetical order withing the groups, so it's easier to identify what is and what isn't imported?

Reply:
Sure, that's reasonable. 

Also, do you have a code-style guide for SKT somewhere? I'd prefer not to learn guidelines one by one.

Reply:
The guidelines are here, but we haven't touched them in a long time so there might be some changes in the meanwhile (eg. in the linked documents): https://github.com/CKI-project/meta/blob/master/CODING.md